include/tvm/relax/struct_info.h

```diff
@@ -180,17 +184,20 @@ class TensorStructInfoNode : public StructInfoNode {
   void VisitAttrs(AttrVisitor* v) {
     v->Visit("shape", &shape);
     v->Visit("dtype", &dtype);
+    v->Visit("vdevice", &vdevice);
     v->Visit("ndim", &ndim);
     v->Visit("span", &span);
   }
 
   bool SEqualReduce(const TensorStructInfoNode* other, SEqualReducer equal) const {
-    return equal(shape, other->shape) && equal(ndim, other->ndim) && equal(dtype, other->dtype);
+    return equal(shape, other->shape) && equal(ndim, other->ndim) &&
+           equal(vdevice, other->vdevice) && equal(dtype, other->dtype);
```

Comment:
This changes the json format, would be good to add a unity updater here https://github.com/apache/tvm/blob/main/python/tvm/ir/json_compact.py#L60 to upgrade the json so when we load older versions they still work

Reply:
[the updater](https://github.com/apache/tvm/pull/15447/files#diff-1a7f4665eeaa8ac3b32bb983b901e52eee4a2e38661c507e1902bd6c46ce7785R60-R72) was added in the new commit. Currently we don't have version management for Unity branch, I used `0.13` as the from_version, `0.14` as the to_version after the upgrade in the updater.